cargo-dist/src/main.rs

```diff
@@ -516,3 +519,57 @@ fn cmd_manifest_schema(
     }
     Ok(())
 }
+
+async fn cmd_update(_config: &Cli, args: &cli::UpdateArgs) -> Result<(), miette::ErrReport> {
+    let mut updater = AxoUpdater::new_for("cargo-dist");
+    // Do we want to treat this as an error?
+    // Or do we want to sniff if this was a Homebrew installation?
+    if updater.load_receipt().is_err() {
+        eprintln!("Unable to load install receipt to check for updates.");
+        eprintln!("If you installed this via `brew`, please `brew upgrade cargo-dist`!");
+        return Ok(());
+    }
+
+    if !updater.check_receipt_is_for_this_executable()? {
+        eprintln!("This installation of cargo-dist wasn't installed via a method that `cargo dist update` supports.");
+        eprintln!("Please upgrade manually.");
```

Comment:
the words update and upgrade are interchanged here- we probably want to keep it consistent